src/handlers.rs

```diff
@@ -342,9 +342,9 @@ impl NetgroupWithName {
         };
 
         if unsafe { setnetgrent(netgroup_name.as_ptr() as *const c_char) } != 1 {
```

Comment:
wait, does it ever return anything but 0 or 1? do we need to check errno?

Reply:
I was curious about that too, this mentioned only 1 or 0:
https://man7.org/linux/man-pages/man3/setnetgrent.3.html